src/open_inwoner/cms/profile/cms_appconfig.py

```diff
@@ -52,3 +53,11 @@ class ProfileConfig(AppHookConfig):
         default=True,
         help_text=_("Designates whether 'questions' section is rendered or not."),
     )
+
```

Comment:
I am not sure if this should be here or in the Profile context view. It's not exactly part of the ProfileConfig model.

Reply:
Makes sense. I moved the function to a separate utils module as it is the same for the profile and contacts view.